### PR TITLE
[WIP] Allow users to include other configuration files.

### PR DESCRIPTION
**Pull request by troessner**
Fixes part 2 of #1411

### CODE REVIEW

**Comment by troessner** [comment]
@mvz should we allow deep nesting of inclusion? wdyt?

File: features/configuration_files/include_other_configuration_files.feature [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      [3]:UnusedParameters: Smelly#x has unused parameter 'y'
+    """
+
+  Scenario: Included file has additional include directives
+
+  Scenario: Other configuration file overwrites configuration in the main configuration
+
+  Scenario: Other configuration file does not exist
+
+  Scenario: Other configuration file contains a syntax error
```
Comment (mvz) [inline features/configuration_files/include_other_configuration_files.feature]: Maybe we can handle the error cases in specs instead?

**Comment by mvz** [comment]
> should we allow deep nesting of inclusion? wdyt?

Yes, I think we should probably allow it, unless it makes things more complicated.

**Comment by troessner** [comment]
Stale, closing for now.

**Pull request closed without merging**